### PR TITLE
Add `.yaml` to accepted yaml extensions in `load_dict_from_file` function

### PR DESCRIPTION
**Pull request by h-mayorquin**
## Motivation

Quick PR as described in the title. I also changed one of the test files to the new accepted extension to test the behavior.

### CODE REVIEW

File: src/nwb_conversion_tools/utils/dict.py [hunk 1]
```diff
@@ -39,9 +39,12 @@ def load_dict_from_file(file_path: FilePathType) -> dict:
     """Safely load metadata from .yml or .json files."""
     file_path = Path(file_path)
     assert file_path.is_file(), f"{file_path} is not a file."
-    assert file_path.suffix in [".yml", ".json"], f"{file_path} is not a valid .yml or .json file."
+    yaml_valid_extensions = [".yml", ".yaml"]
+    file_is_valid_yaml = file_path.suffix in yaml_valid_extensions
+    file_is_valid_json = file_path.suffix == ".json"
+    assert file_is_valid_yaml or file_is_valid_json, f"{file_path} is not a valid .yml or .json file."
```
Comment (bendichter) [inline src/nwb_conversion_tools/utils/dict.py:45]:
```suggestion
    assert file_path.suffix in (".json", ".yml", ".yaml"), f"{file_path} is not a valid .yml or .json file."
```

File: src/nwb_conversion_tools/utils/dict.py [hunk 2]
```diff
@@ -39,9 +39,12 @@ def load_dict_from_file(file_path: FilePathType) -> dict:
...
 
-    if file_path.suffix == ".yml":
+    if file_path.suffix in yaml_valid_extensions:
```
Comment (bendichter) [inline src/nwb_conversion_tools/utils/dict.py:47]:
```suggestion
    if file_path.suffix in (".yml", ".yaml"):
```

**Comment by h-mayorquin** [comment]
> @h-mayorquin Is this one familiar to you? https://github.com/catalystneuro/nwb-conversion-tools/runs/6566203521?check_suite_focus=true

No, it also did not appear the first time.

**Comment by h-mayorquin** [comment]
@CodyCBakerPhD 
There seems to be an issue when the cached version of the gin data is not being used.

**Comment by CodyCBakerPhD** [comment]
> @CodyCBakerPhD There seems to be an issue when the cached version of the gin data is not being used.

Can you point me to that exact error? I'm having trouble finding something about that

**Comment by CodyCBakerPhD** [comment]
Are you able to track the Movie issue down to a specific test? Is it random again or deterministic this time? If you can pin it down we can just disable it and raise an issue if needed

**Review by CodyCBakerPhD — APPROVED** [review]

**Pull request merged**